Updates submodules for ion-functions

lukecampbell opened the pull request:
Previous HEAD position was 8a8b432... Merge pull request #68 from
lukecampbell/loclrng-r2
HEAD is now at 8aae5b4... Merge pull request #69 from
lukecampbell/fillvals-r2

lukecampbell commented:
@jamie-cyber1 Ready to merge at your leisure.

jamie-cyber1 commented on extern/marine-integrations:
```diff
@@ -0,0 +1 @@
+Subproject commit 99a14c98ccb82ac41b879ad1ad7d37d2f7354e24
```
Why mi pointer here?

lukecampbell replied:
That's an EXCELLENT question.

lukecampbell committed:
Updates ion-functions submodule

Previous HEAD position was 8a8b432... Merge pull request #68 from
lukecampbell/loclrng-r2
HEAD is now at 8aae5b4... Merge pull request #69 from
lukecampbell/fillvals-r2

lukecampbell commented:
So, when I updated the submodule I did a `git add extern/` before committing and apparently we don't have a marine-integrations submodule anymore but I still have a folder (with a git reference) so it got roped in with the commit, I just force-pushed a proper commit and it is ready to be reviewed.

jamie-cyber1 commented:
Looks good.
